families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,398 @@
+      parameters:
+        - name: identifier
+          in: query
+          type: string
+          description: A partial identifier to filter Records by
+        - $ref: "#/parameters/count"
+        - $ref: "#/parameters/min"
+        - $ref: "#/parameters/max"
+        - $ref: "#/parameters/sort"
+      responses:
+        200:
+          description: Successfully retrieved Records
+          schema:
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: "#/definitions/Record"
+              link:
+                $ref: "#/definitions/Link"
+              paging:
+                $ref: "#/definitions/Paging"
+        400:
+          $ref: "#/responses/400BadRequest"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+  /records/{record_id}:
+    parameters:
+      - $ref: "#/parameters/record_id"
+    get:
+      summary: Fetches a particular Record.
+      responses:
+        200:
+          description: Successfully fetched Record
+          schema:
+            properties:
+              data:
+                $ref: "#/definitions/Record"
+              link:
+                $ref: "#/definitions/Link"
+        400:
+          $ref: "#/responses/400BadRequest"
+        404:
+          $ref: "#/responses/404NotFound"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+
+responses:
+  400BadRequest:
+    description: Request was malformed
+    schema:
+      $ref: "#/definitions/Error"
+  404NotFound:
+    description: Identifier did not match any resource
+    schema:
+      $ref: "#/definitions/Error"
+  500ServerError:
+    description: Something went wrong within the validator
+    schema:
+      $ref: "#/definitions/Error"
+  503ServiceUnavailable:
+    description: API is unable to reach the validator
+    schema:
+      $ref: "#/definitions/Error"
+
+parameters:
+  agent_id:
+    name: agent_id
+    in: path
+    type: string
+    required: true
+    description: Agent identifier, the public key the agent uses.
+  record_id:
+    name: record_id
+    in: path
+    type: string
+    required: true
+    description: "Record identifier, the serial number or natural identifier
+      of the item (material) being tracked."
+  wait:
+    name: wait
+    in: query
+    type: integer
+    description: A time in seconds to wait for commit
+  count:
+    name: count
+    in: query
+    type: integer
+    default: 1000
+    description: Number of items to return
+  min:
+    name: min
+    in: query
+    type: string
+    description: Id or index to start paging (inclusive)
+  max:
+    name: max
+    in: query
+    type: string
+    description: Id or index to end paging (inclusive)
+  sort:
+    name: sort
+    in: query
+    type: string
+    description: |
+      Comma-separated keys to sort a list by:
```

Comment:
Many of these sort parameters don't make sense for this API - they shouldn't be referencing headers and signing keys, necessarily, but fields on the objects returned by the call.